remme/tp/consensus_account.py

```diff
@@ -125,82 +120,71 @@ def _send_reward(self, context, public_key, pb_payload):
             raise InvalidTransaction('Bet for address not found.')
         else:
             obligatory_payments = consensus_account.obligatory_payments
-
-            # TODO: Take from here in the future
-            # block_cost = consensus_account.block_cost
-            block_cost = zero_account.balance
-
+            block_cost = consensus_account.block_cost
 
         min_stake, unfreeze_bonus, max_share, min_share = self._get_share_data(context)
 
         reputational = real_to_client_amount(node_account.reputation.unfrozen + node_account.reputation.frozen)
 
-        real_reward = obligatory_payments + block_cost + bet
-        reward = real_to_client_amount(real_reward)
+        reward = real_to_client_amount(obligatory_payments + block_cost + bet)
```

Comment:
real_to_client divides value by 10000. 

So we have obligatory_payments = 2.0000, block_cost = 10.0000, bet = 10.0000
Then we add them to each others that's equal to 22.0000

Why we then divide 22.0000 by 10000? 
https://github.com/Remmeauth/remme-core/blob/dev-consensus/remme/shared/utils.py#L45